src/main/java/Duke.java

```diff
@@ -1,10 +1,106 @@
+import java.util.Scanner;
+
 public class Duke {
-    public static void main(String[] args) {
+
+    private static Tasks[] list_of_tasks = new Tasks[101];
```

Comment:
Consider other data structures, apart from a fixed size array, to make your product more scalable, especially since we are already approaching recess week :)